src/app/modules/connect-to-wallet/connect-buttons/connect-buttons.component.spec.ts

```diff
@@ -5,16 +5,20 @@ import { DebugElement } from '@angular/core';
 import { getElement } from '@tests';
 import { ProviderType } from 'iam-client-lib';
 import { MetamaskProviderService } from '../../../shared/services/metamask-provider/metamask-provider.service';
+import { EkcSettingsService } from '../ekc-settings/services/ekc-settings.service';
 
 describe('ConnectButtonsComponent', () => {
   let component: ConnectButtonsComponent;
   let fixture: ComponentFixture<ConnectButtonsComponent>;
   let hostDebug;
   let connectToSpy;
-  const metamaskProviderServiceSpy = jasmine.createSpyObj(
+  let metamaskProviderServiceSpy = jasmine.createSpyObj(
```

Comment:
why let?

Reply:
Yeah, eslint complains on this. Already changed.